pydoctor/astbuilder.py

```diff
@@ -470,7 +472,8 @@ def _handleClassVar(self,
         assert isinstance(cls, model.Class)
         if not _maybeAttribute(cls, name):
             return
-        obj: Optional[model.Attribute] = cls.contents.get(name)
+        # Class variables can only be Attribute, so it's OK to cast
+        obj = cast(Optional[model.Attribute], cls.contents.get(name))
```

Comment:
Should we use `assert` here too ?

Reply:
I believe you should be able to assert before this line, and then the cast should be unnecessary, because mypy tries to narrow types when `isinstance` assertions are made.